spring-pulsar/src/main/java/org/springframework/pulsar/aot/PulsarRuntimeHints.java

```diff
@@ -56,7 +57,7 @@ public void registerHints(RuntimeHints hints, @Nullable ClassLoader classLoader)
 		// through Pulsar client.
 		Stream.of(HashSet.class, LinkedHashMap.class, TreeMap.class, Authentication.class,
 				AuthenticationDataProvider.class, SecretsSerializer.class, PulsarAdminBuilderImpl.class,
-				OffloadProcessStatusImpl.class, Commands.class).forEach(
+				PulsarClientImplementationBindingImpl.class, OffloadProcessStatusImpl.class, Commands.class).forEach(
```

Comment:
This covers both imperative and reactive cases since reactive leans heavily on the imperative client.